FAQ.md

```diff
@@ -36,9 +36,12 @@ to integrate with container ecosystems.
 
 ### What processors does Firecracker support?
 
-The Firecracker VMM is built to be processor agnostic. Intel processors are
-supported for production workloads. Support for AMD and Arm processors is in
-developer preview.
+The Firecracker VMM is built to be processor agnostic. Intel (Skylake and Cascade
+Lake) and AMD Zen2 processors are supported for production workloads. Support for
+Arm processors is in developer preview.
```

Comment:
Firecracker support for Arm CPUs is released as developer preview.

Reply:
done